group_vars/all.sample

```diff
@@ -132,7 +131,7 @@ dummy:
 #ceph_stable_ice_password: # htaccess password
 
 # ENTERPRISE VERSION RED HAT STORAGE (from 1.3)
-# This version is only supported on RHEL >= 7.1
```

Comment:
do you mind updating `roles/ceph-common/defaults/main.yml` to reflect that?
then run `generate_group_vars_sample.sh`.
Thanks!